app/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,17 @@
+<resources>
+
+    <string name="app_name">Filmes Populares</string>
+    <string name="app_generic_connection_error">Ops! No momento não conseguimos buscar os dados!</string>
+    <string name="app_generic_no_connection_error">Ops! Certifique-se que seu device tem conexão, e tente novamente!
+    </string>
+
+    <!-- Menu -->
+    <string name="menu_search">Pesquisar</string>
+
+    <!-- Detail -->
+    <string name="fragment_detail_open_trailer_error">Ops! aconteceu algo de errado!</string>
+    <string name="fragment_detail_open_trailer_release_date">Lançamento %s</string>
+    <string name="fragment_detail_open_trailer_release_language">Linguagem %s</string>
+    <string name="fragment_detail_open_trailer_release_popularity">Popularidade %s</string>
```

Comment:
What are your concerns when dealing with internationalisation and strings that have arguments (such as `%s`)?

Reply:
Hello, this is something that the API will handle, because the value that i'm passing in the param is coming from the API, and this information is localized by the value i'm passing according to the language of the device.